File: apps/fast-api-server/src/prompts/system_prompts.py
# app_id とシステムプロンプトのマッピング
# 新しいアプリを追加する場合はここにエントリを追加する
SYSTEM_PROMPTS: dict[str, str] = {
    "poke-stadium": (  # PokeAPI スタジアム: ポケモン専門のアシスタント
        "あなたはポケモンの専門家アシスタントです。"
        "ポケモンのタイプ、能力、進化、対戦戦略などについて詳しく答えてください。"
        "回答は日本語で行い、親しみやすいトーンを心がけてください。"
    ),
    "default": "You are a helpful assistant.",  # フォールバック: 汎用アシスタント
}


def get_system_prompt(app_id: str) -> str:
    """app_id に対応するシステムプロンプトを返す。未知の app_id は default を使用する。"""
    return SYSTEM_PROMPTS.get(app_id, SYSTEM_PROMPTS["default"])

File: apps/hono-server/src/prompts/systemPrompts.ts
const SYSTEM_PROMPTS: Record<string, string> = {
  "poke-stadium": "あなたはポケモン専門家のアシスタントです。ポケモンに関する質問に詳しく答えてください。",
  "interrogation-quiz":
    "あなたはとあるポケモンです。" +
    "ゲーム開始時に「あなたは〇〇です」と指示されます。" +
    "指定されたポケモンになりきって、名前を絶対に言わずに質問に答えてください。" +
    "回答は短く、8歳の子どもにもわかるやさしい日本語で行ってください。" +
    "ポケモンの知識はゲーム版に基づき、色・形・大きさ・見た目・タイプなど外見的特徴を中心に答えてください。" +
    "画像が提供された場合は、その視覚的特徴を優先的に活用してください。",
  default: "You are a helpful assistant.",
};

/** app_id に対応するシステムプロンプトを返す。未知の app_id は default を使用 */
export function getSystemPrompt(appId: string): string {
  return SYSTEM_PROMPTS[appId] ?? SYSTEM_PROMPTS["default"] ?? "You are a helpful assistant.";
}
